src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,9 +5,19 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private double bladeWidth;
+    private double bladeCapacity;
```

Comment:
Don't forget about access modifiers for your custom fields in `Bulldozer` class. It's good practice to use private access modifiers for fields and provide public getters and setters for accessing them.